fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/directory/FDBDirectoryLockTest.java

```diff
@@ -35,7 +43,21 @@
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 @Tag(Tags.RequiresFDB)
-class FDBDirectoryLockTest extends FDBDirectoryBaseTest {
+class FDBDirectoryLockTest {
```

Comment:
I ran this test in a mode that tracked all contexts, and it found some unclosed exceptions. This is currently not failing, because the opened contexts aren't being tracked (or I think that's the issue). The agility context was creating contexts it didn't open, though, at least on the branch I had. It's possible this has been fixed on a newer branch.

Reply:
AgilityContext should be properly closing all contexts, at least if it is closed properly, which this class might not do.